kuma/static/styles/components/content.scss

```diff
@@ -227,9 +227,12 @@ We cannot put these styles in the base/elements/ files because they don't apply
     typography
     ====================================================================== */
 
-    /* don't allow empty paragraphs by CKEditor */
+    /*
+     * Don't allow empty paragraphs by CKEditor.
+     * But do allow empty divs created by the Prism syntax highlighter.
+     */
     p:empty,
-    div:empty {
+    div:not(.line-highlight):empty {
```

Comment:
This change makes line highlighting work again. See http://beta.mdn.localhost:8000/en-US/docs/Web/CSS/gap#Examples for an example.  Line 5 of the CSS example should be highlighted, but it isn't without this change.